resources/lang/de/users.php

```diff
@@ -0,0 +1,270 @@
+                        ],
+
+                        'vote' => [
+                            'give' => ':amount durch erhaltene Stimmen im Post :post erhalten',
+                            'reset' => ':amount durch verlorene Stimmen im Post :post verloren',
+                        ],
+
+                        'recalculate' => [
+                            'give' => ':amount durch Neuberechnung der Stimmen in :post erhalten',
+                            'reset' => ':amount durch Neuberechnung der Stimmen in :post verloren',
+                        ],
+                    ],
+
+                    'forum_post' => [
+                        'give' => ':amount von :giver für einen Post in :post erhalten',
+                        'reset' => 'Kudosu von :giver im Post :post zurückgesetzt',
+                        'revoke' => 'Kudosu von :giver im Post :post verwehrt',
+                    ],
+                ],
+            ],
+            'me' => [
+                'title' => 'ich!',
+            ],
+            'medals' => [
+                'empty' => 'Dieser Nutzer hat noch keine erhalten. ;_;',
+                'title' => 'Medaillen',
+            ],
+            'recent_activity' => [
+                'title' => 'Neulich',
+            ],
+            'top_ranks' => [
+                'best' => [
+                    'title' => 'Beste Performance',
+                ],
+                'empty' => 'Noch keine Performance-Rekorde. :(',
+                'first' => [
+                    'title' => 'Erster Platz',
+                ],
+                'pp' => ':amountpp',
+                'title' => 'Ränge',
+                'weighted_pp' => 'gewichtet: :pp (:percentage)',
+            ],
+        ],
+        'info' => [
+            'interests' => 'Interessen',
+            'lastfm' => 'Last.fm',
+            'location' => 'Aktueller Standort',
+            'occupation' => 'Beschäftigung',
+            'skype' => 'Skype',
+            'twitter' => 'Twitter',
+            'website' => 'Webseite',
+        ],
+        'page' => [
+            'description' => '<strong>me!</strong> ist ein persönlicher Bereich auf deinem osu!-Profil, den du nach deinem Belieben anpassen kannst.',
+            'edit_big' => 'me! bearbeiten',
+            'placeholder' => 'Seiteninhalt hier eingeben',
+            'restriction_info' => "Für dieses Feature musst du ein <a href='".route('store.products.show', 'supporter-tag')."' target='_blank'>osu!-Supporter</a> sein.",
+        ],
+        'rank' => [
+            'country' => 'Länderrang im Modus :mode',
+            'global' => 'Globaler Rang im Modus :mode',
+        ],
+        'stats' => [
+            'hit_accuracy' => 'Genauigkeit',
+            'level' => 'Level :level',
+            'maximum_combo' => 'Höchste Combo',
+            'play_count' => '<Play>anzahl',
+            'ranked_score' => 'Punktzahl auf Ranglisten',
+            'replays_watched_by_others' => 'Replays Watched by Others',
```

Comment:
`Wiederholungen von anderen angeschaut`

Reply:
Hier ging es um den Begriff "Replay".. etwas schwierig. "Wiederholung" ist das Erste, was in den Sinn kommt. Überleg ich mir noch mal